jupyter/Scale Output Example.ipynb

```diff
@@ -0,0 +1,182 @@
+     }
+    },
+    "systemlink": {
+     "namespaces": [
+      "ni-testmanagement"
+     ],
+     "outputs": [
+      {
+       "display_name": "Input plus fraction",
+       "id": "scalar_output",
+       "type": "scalar"
+      }
+     ],
+     "parameters": [
+      {
+       "display_name": "Number",
+       "id": "numeric_param",
+       "type": "number"
+      }
+     ],
+     "version": 2
+    },
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Parameters\n",
+    "numeric_param = 1 # default value used when input is not provided by Dashboard\n",
+    "\n",
+    "result = [] # container for all of the notebooks results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "output = int(numeric_param) + uniform(0,1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Results\n",
+    "\n",
+    "Scalar results are appended with a json map that corresponds to the json in the cell metadata described above. The \"type' and 'id' fields should match the cell metadata values, and the 'value' field should contain the scalar result. The config block doesn't correspond to anything in the cell metadata."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Scalar result needs to be added to the result list and glued to the scrapbook\n",
+    "# The json descriptor should match the descriptor in the outputs section in the parameters cell metadata.\n",
+    "result.append({\n",
+    "    'type': 'scalar',\n",
+    "    'id': 'scalar_output',\n",
+    "    'config': {\n",
+    "        'title': 'Scalar Output Title'\n",
+    "    },\n",
+    "    'value': output\n",
+    "})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/scrapbook.scrap.json+json": {
+       "data": [
+        {
+         "config": {
+          "title": "Scalar Output Title"
+         },
+         "id": "scalar_output",
+         "type": "scalar",
+         "value": 1.322663202537357
+        }
+       ],
+       "encoder": "json",
+       "name": "result",
+       "version": 1
+      }
+     },
+     "metadata": {
+      "scrapbook": {
+       "data": true,
+       "display": false,
+       "name": "result"
+      }
+     },
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "[{'type': 'scalar',\n",
+       "  'id': 'scalar_output',\n",
+       "  'config': {'title': 'Scalar Output Title'},\n",
+       "  'value': 1.322663202537357}]"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Gluing the result list to the scrapbook is what actually allows you to bind the result to the numeric Dashboard tile.\n",
```

Comment:
Note, you don't really have to do anything for this: Technically the styleguide recommends using 100 character lines, but that rule is particularly suited towards Python source files and makes less sense in notebooks. I don't think the conventions were really ever fully designed for notebooks because other rules don't apply whatsoever.
https://ni.github.io/python-styleguide/#F-1-2

Line length is fragmented in python and others agree on 120 (and this line is 119), so it might be fine to deviate here and keep it, since there's no reason to have such a short line length limit in a notebook outside of a .py file. Just something to be aware of.